server/controllers/manage/premises/premisesController.test.ts

```diff
@@ -51,39 +54,33 @@ describe('V2PremisesController', () => {
   })
 
   describe('show', () => {
-    const mockSummaryAndPlacements = async (
```

Comment:
This was replaced with the more conventional approach of setting up mocks in a `beforeEach` block, then setting the request and calling the handler in each test. 

Originally, I had a use case where I needed to replace the mocked returned value of one of the `premisesService` response that is set there, and this construct rendered this impossible. As it turns out, there doesn't seem to be any such use case leftover, but I've left it as it now is as I think it seems just as clear.